appeals/web/src/server/lib/string-utilities.js

```diff
@@ -6,3 +6,20 @@
 export const stringContainsDigitsOnly = (str) => {
 	return !!str.trim().match(/^\d+$/);
 };
+
+/**
+ *
+ * @param {string} str
+ * @returns {string}
+ */
+export const capitalize = (str) => {
```

Comment:
You can use the lodash function for this instead (I think): `import { capitalize } from 'lodash-es';`

Reply:
Not suitable here. `capitalize` from lodash modifies the rest of the srting. So 'LPA_questionnaire' becomes 'Lpa_questionnaire'